README.md

```diff
@@ -58,9 +58,15 @@ If you're curious about the edge cases that `gosu` handles, see [`Dockerfile.tes
 
 ## Alternatives
 
-### `su-exec`
+### `ncopa/su-exec`
 
-As mentioned in `INSTALL.md`, [`su-exec`](https://github.com/ncopa/su-exec) is a very minimal re-write of `gosu` in C, making for a much smaller binary, and is available in the `main` Alpine package repository.
+As mentioned in `INSTALL.md`, [`ncopa/su-exec`](https://github.com/ncopa/su-exec) is a very minimal re-write of `gosu` in C, making for a much smaller binary, and is available in the `main` Alpine package repository.
+
+The ncopa's su-exec have critical bugs (or features), it will use root permission silently if invalid user-spec given !
+
+### `songdongsheng/su-exec`
+
+[`songdongsheng/su-exec`](https://github.com/songdongsheng/su-exec) is another very minimal re-write of `gosu` in C, making for a much smaller binary, and safer than the [`ncopa/su-exec`](https://github.com/ncopa/su-exec).
```

Comment:
You're definitely going to have to qualify this a bit; what do you mean by "safer"?

Have you run our existing test suite against it to verify compatibility?

Did you perhaps consider instead collaborating with ncopa on making his implementation better before creating your own in the same language with the same apparent set of goals? (He's a nice guy, and I'm sure would be open to improvement if there's something concrete.)